app/views/rooms/show.html.haml

```diff
@@ -0,0 +1,10 @@
+.container
+  %h2= @room.name
+  .room-description= @room.description
+  .room-capacity= "#{@room.reviewees.count} / #{@room.capacity}"
```

Comment:
__[HAML-Lint]__

`= "..."` should be rewritten as `...`


<!-- Comment by [SideCI](https://sideci.com) -->